src/command_modules/azure-cli-resource/setup.py

```diff
@@ -27,6 +27,7 @@
 DEPENDENCIES = [
     'azure-mgmt-resource==1.0.0rc1',
     'azure-cli-core',
+    'azure-mgmt-authorization'
```

Comment:
Should pin to a version for the SDKs.
We may consider not pinning once all mgmt SDKs support multiple API versions but until then, I would pin.
Looks like it's 0.30.0rc6 that's used elsewhere.